### PR TITLE
Making diktat consistent with own code style (rules.kdoc + diktat-rules tests)

### PR DESCRIPTION
**Pull request by petertrr**
### What's done:
* Tried to make just 4 files (diktat-rules -> rules.kdoc) consistent with code style. Tried to use autocorrect and opened #480 , #481, #482 , #483, #484 while doing this. Only 2639 more errors to be completely consistent...
* Applied code style to diktat-rules tests. Opened #488, #489, #490

### CODE REVIEW

File: diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/constants/Warnings.kt [hunk 1]
```diff
@@ -6,6 +6,8 @@ import org.cqfn.diktat.common.config.rules.isRuleEnabled
 import org.cqfn.diktat.ruleset.utils.hasSuppress
 import org.jetbrains.kotlin.com.intellij.lang.ASTNode
 
+typealias EmitType = ((offset: Int, errorMessage: String, canBeAutoCorrected: Boolean) -> Unit)
```
Comment (akuleshov7) [inline diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/constants/Warnings.kt:9]: fair enough :)

File: diktat-rules/src/test/kotlin/org/cqfn/diktat/util/TestUtils.kt [hunk 2]
```diff
@@ -1,49 +1,80 @@
...
-                .hasSize(expectedLintErrors.size)
                 .allSatisfy { actual ->
                     val expected = expectedLintErrors[this.indexOf(actual)]
                     SoftAssertions.assertSoftly {
-                        it.assertThat(actual.line).`as`("Line").isEqualTo(expected.line)
-                        it.assertThat(actual.col).`as`("Column").isEqualTo(expected.col)
-                        it.assertThat(actual.ruleId).`as`("Rule id").isEqualTo(expected.ruleId)
-                        it.assertThat(actual.detail).`as`("Detailed message").isEqualTo(expected.detail)
-                        it.assertThat(actual.canBeAutoCorrected).`as`("Can be autocorrected").isEqualTo(expected.canBeAutoCorrected)
+                        it.assertThat(actual.line)
```
Comment (akuleshov7) [inline diktat-rules/src/test/kotlin/org/cqfn/diktat/util/TestUtils.kt:34]: looks a little bit strange for short statements, but looks good for long. Where is the balance?

**Review by akuleshov7 — APPROVED** [review]
lgtm

File: diktat-rules/src/test/kotlin/org/cqfn/diktat/util/TestUtils.kt (hunk 2 above)
Comment (petertrr) [reply diktat-rules/src/test/kotlin/org/cqfn/diktat/util/TestUtils.kt:34]: It actually even isn't compliant with our current code style: we are forcing a line break after `it`, but I thought it looks too ugly. I'll add an issue about making this rule prettier and maybe taking into account text length.

**Pull request merged**